### PR TITLE
Add pg_shard deprecation message suggesting Citus extension

### PR DESCRIPTION
**Pull request by begriffs**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## Deprecation Notice
+
+The new open source [Citus extension](https://github.com/citusdata/citus)
+for PostgreSQL provides a superset of the functionality of `pg_shard`.
```
Comment (ozgune) [inline doc/README.md]: Minor.

`for PostgreSQL provides a superset` --> `for PostgreSQL natively integrates pg_shard and provides a superset of its functionality`, or a better wording of this to signal that Citus integrates pg_shard in full?

File: doc/README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+## Deprecation Notice
+
+The new open source [Citus extension](https://github.com/citusdata/citus)
```
Comment (jasonmp85) [inline doc/README.md]: Hyphenate _open-source_.

File: doc/README.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+for PostgreSQL natively integrates `pg_shard` and provides a superset
+of its functionality.  We’ll not be making any more `pg_shard`
+releases and the project is entering end-of-life status.
+
+We have migration steps to help existing customers perform a drop-in
+replacement. [Contact
+us](mailto:engage@citusdata.com?subject=pg_shard+Migration) for
+more information, or feel free to drop in to our [IRC
```
Comment (jasonmp85) [inline doc/README.md]: Replace _drop in_ with _stop by_. The proximity of the phrase to _drop-in replacement_ felt weird to me.

**Comment by jasonmp85** [comment]
OK, made the small fixes myself. Will merge after status checks pass.

**Pull request merged**